src/cur-prot/annotate-speeches.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
+"""
+Create IDs for speeches. Add speech IDs to metadata.
+"""
+from lxml import etree
+from pyriksdagen.args import (
+    fetch_parser,
+    impute_args,
+)
+from pyriksdagen.io import (
+    parse_tei,
+    write_tei
+)
+from pyriksdagen.utils import (
+    elem_iter,
+    get_formatted_uuid,
+)
+from tqdm import tqdm
+from trainerlog import get_logger
+import os
+
+
+
+
+logger = get_logger(name="Trainer Log", level=os.environ.get("LOGLEVEL", None))
```

Comment:
Give a name that describes the scipts somewhat, eg. "annotate", or just "logger". The level stuff should also work as a default from the package, if not file an issue there. So the level argument can be deleted.